src/test/resources/application.yml

```diff
@@ -157,7 +157,13 @@ ccd:
   data-store:
     api:
       url: ${CCD_DATA_STORE_API_BASEURL:http://localhost:4452}
+  jurisdictionid: DIVORCE
+  casetype: NO_FAULT_DIVORCE
 
+core_case_data:
+  api:
+    url: ${CCD_DATA_STORE_API_BASEURL:http://localhost:4452}
```

Comment:
`CaseRoleClient.java` uses below added by upstream
```
ccd:
  data-store:
    api:
      url: ${CCD_DATA_STORE_API_BASEURL:http://localhost:4452}
```
and `CoreCaseDataApi.java`  uses
```
core_case_data:
  api:
    url: ${CCD_DATA_STORE_API_BASEURL:http://localhost:4452}
```
as a separate PR I think we should make it consistent